issues-policy.md

```diff
@@ -0,0 +1,84 @@
+# GitHub issues process and policy
+
+The goals of the process are:
+
+1. Ensure errors or omissions in our docs are not blocking customer success.
+1. Be responsive to customer feedback and concerns.
+1. Continually improve the customer experience.
+1. Learn more about customer experiences through dialogue in the open on challenges and solutions.
+
+The process uses two stages to ensure responsiveness while prioritizing work. The initial stage diagnoses and triages the issue. The second stage resolves the issue. When an issue is both easy and urgent, the two stages may be combined.
+
+The process involves two time boxed tasks:
+
+- Each team member spends up to 1/2 hour each business day [classifying incoming issues](#diagnosis-phase), including initial response. This ensures we are responsive on new issues.
+- Each team member spends up to 1.5 days per sprint [updating documents](#resolution-phase) fixing customer generated GitHub issues.
```

Comment:
fixing customer generated --> to address customer-generated

Reply:
Do we explain what a sprint is somewhere?

Reply:
In another place, this PR suggests more uniform frequency: "four hours per week". Then, notion of sprint is not necessary.

Reply:
in places we talk about fitting something into current or later sprint, so it would be useful for people to understand that "sprint" for us means a three-week period, even if we could link out to a common definition of the word.

Reply:
I removed the references to "sprint" and translated to weeks.